src/doc/reference/src/visibility-and-privacy.md

```diff
@@ -40,7 +40,10 @@ With the notion of an item being either public or private, Rust allows item
 accesses in two cases:
 
 1. If an item is public, then it can be used externally through any of its
-   public ancestors.
+   accessible ancestors (either by anyone that has an access to an ancestor
+   in the case of a chain of public ancestors,
+   or by the siblings of a private ancestor, in the case of a chain of public
+   ancestors terminating into a private ancestor).
```

Comment:
If you or someone else have better ideas about the wording, I'd appreciate better suggestions.